### PR TITLE
Add files via upload

### LINKED ISSUE #411: Add code for Trie Data Structure
# Description
Add code for Trie DataStructure

# Additional context
Add any other context, links references or screenshots about the feature request here.

# Important Checklist 
-   [x] I have read the contribution guidelines before making this issue.
-   [x] I have verified this issue program is not already implemented
-   [x] I have not pushed this code anywhere else (even my repo)

# Languages and Assignees:
-   [x] Python: @namitamutha 
-   [x] Java: @AK271 
-   [ ] C: @
-   [x] C++: @Preetesh21

### PR DESCRIPTION
**Pull request by AK271**
# Description

Java code for Trie Node implementation

Fixes #411  

## Type of change
Added Java file

**Please delete options that are not relevant**

-   [x] New feature (non-breaking change which adds functionality)
-   [x] This change requires a documentation update

# Checklist:

-   [x] I have read the contribution guidelines before making this PR.
-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented in my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation (Language folder or any other readme instructed to)
-   [x] My changes generates no new warnings
-   [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Java/Trie_Implementation.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
+import java.util.Scanner;
```
Comment (kaustubhgupta) [inline Java/Trie_Implementation.java:1]: Update java folder readme

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
@AK271 Edit this PR title

**Comment by AK271** [comment]
I have updated the java folder README.md and added the program and link

**Review by YashSaxena9 — APPROVED** [review]
Implementation seems correct for Trie, thank you for your hard work :)

**Comment by YashSaxena9** [comment]
PR accepted for merge :)

**Comment by AK271** [comment]
Thank you!

**Pull request merged**